### ISSUE TITLE
2fa tests passing in local but failing in CI

### ISSUE THREAD

**Issue by okaycj** [lookit/lookit-api#1055]
**TL;DR**: A few tests to prove the new authenticated redirect works as expected fail in CI. 

**Narrative**
These tests where passing in the local environment, but fail in CI.  This caused a weeks worth of research to figure out why and at the end it seems to be related to the ENVIRONMENT value set in the .env file.  When this value is unset, the tests seem to fail.  
These tests in question are marked as "skip" and in the skip note will be this issue number.  

**Acceptance Criteria**
These tests need to be refactored to pass in CI.  

**Implementation Notes**
These tests are failing due to something with either 2fa or revproxy.
